### PR TITLE
A31: update xds field names to reflect recent changes

### PR DESCRIPTION
**Pull request by dfawley**
Ref: https://github.com/envoyproxy/envoy/pull/13247

### CODE REVIEW

File: A31-xds-timeout-support-and-config-selector.md [hunk 1]
```diff
@@ -164,38 +164,38 @@ support fields use to control the first timer, but not the second timer.
 Specifically, the following xDS timeout-related settings will be supported:
 
 - [`RouteAction.max_stream_duration.max_stream_duration`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L775)
-- [`RouteAction.max_stream_duration.grpc_max_timeout`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L781)
+- [`RouteAction.max_stream_duration.grpc_timeout_header_max`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L781)
```
Comment (markdroth) [inline A31-xds-timeout-support-and-config-selector.md:167]: You'll want to update the link as well.

File: A31-xds-timeout-support-and-config-selector.md [hunk 2]
```diff
@@ -164,38 +164,38 @@ support fields use to control the first timer, but not the second timer.
...
 - [`max_stream_duration`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/core/v3/protocol.proto#L95) from the [`HttpConnectionManager.common_http_options`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto#L289) as a default if `RouteAction.max_stream_duration.max_stream_duration` is unset.
 
 gRPC will not support:
 
 - [`RouteAction.timeout`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L952).
   This field begins the timeout counter from a point in time incompatible with gRPC's timeout model.
-- [`RouteAction.max_stream_duration.grpc_timeout_offset`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L789).
+- [`RouteAction.max_stream_duration.grpc_timeout_header_offset`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L789).
```
Comment (markdroth) [inline A31-xds-timeout-support-and-config-selector.md:174]: Same here.

File: A31-xds-timeout-support-and-config-selector.md (hunk 1 above)
Comment (dfawley) [reply A31-xds-timeout-support-and-config-selector.md]: Argh! Good catch, thanks.

File: A31-xds-timeout-support-and-config-selector.md (hunk 2 above)
Comment (dfawley) [reply A31-xds-timeout-support-and-config-selector.md]: Done.

**Review by markdroth — APPROVED** [review]
Thanks for the update!

**Pull request merged**